tests/test_vlan.py

```diff
@@ -186,3 +186,63 @@ def test_MultipleVlan(self, dvs, testlog):
         tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_VLAN")
         vlan_entries = [k for k in tbl.getKeys() if k != dvs.asicdb.default_vlan_id]
         assert len(vlan_entries) == 0
+
+    @pytest.mark.parametrize("test_input, expected", [
+        (["Vla",  "2"], 0),
+        (["VLAN", "3"], 0),
+        (["vlan", "4"], 0),
+        (["Vlan", "5"], 1),
+    ])
+    def test_AddVlanWithIncorrectKeyPrefix(self, dvs, testlog, test_input, expected):
+        self.setup_db(dvs)
+        vlan = test_input[1]
+
+        # create vlan
+        tbl = swsscommon.Table(self.cdb, "VLAN")
+        fvs = swsscommon.FieldValuePairs([("vlanid", vlan)])
+        tbl.set(test_input[0] + vlan, fvs)
+        time.sleep(1)
+
+        # check asic database
+        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_VLAN")
+        vlan_entries = [k for k in tbl.getKeys() if k != dvs.asicdb.default_vlan_id]
+        assert len(vlan_entries) == expected
+
+        if len(vlan_entries) == 0:
+            # check error log
+            result = dvs.runcmd("cat /var/log/syslog")
+            errPos = result[1].find("Invalid key format. No 'Vlan' prefix: " + test_input[0] + test_input[1])
```

Comment:
can you use to get a marker, and then search after the marker? It helps in case there could be multiple such syslog log messages in the syslog.

marker = dvs.add_log_marker()

exmaple code in test_warm_reboot.py, line 613.

Reply:
OK, I will use this method to search syslog log message on the new PR for master branch.